openshift/templates/api/api-deploy.yaml

```diff
@@ -230,7 +230,7 @@ objects:
           - $namespace=${NAMESPACE_NAME}-${TAG_NAME}
       destination:
         - - role=db
-          - app=${APP_NAME}${SUFFIX}
+          - app=${APP_NAME}
```

Comment:
The wallet (db) config has a suffix.  It's just blank at the moment.  You should define a `${DB_SUFFIX}` for this purpose.

Reply:
Since we are consolidating to have just one shared db pod across multiple issuer I figured I could leave the suffix out

Reply:
The template for the wallet supports a suffix.  Any templates depending on that resource should also support the suffix.  Just because we're only deploying a single instance of the applications into a given environment now, does not preclude us from deploying other instances of the same application components into the same environments for some completely different purpose.  When this happens we'll need to provide specific values for the suffixes to keep things separated and ensure the correct services get wired together correctly.

Reply:
added DB_SUFFIX to DB_APP_NAME